### DISCUSSION TITLE
DEBUG do VS Code.... Não para nos breakpoints!

### DISCUSSION THREAD

**Discussion by danilobicas-unoeste** [balta-io/forum#640] [Tira Dúvidas] [answered]
Olá.
Eu executo meu projeto wasm com o debug. OK!
MAS a execução não para nos breakpoints!!!! Eles ficam todos em vermelho mas quando ele abre o browser os breakpoints ficam somente com um círculo (some o vermelho) e não para nestes breakpoints. Importante dizer que ele não para em nenhum... ocorre o mesmo comportamento em todos. O dotnet watch run, dotnet build, dotnet run funcionam também sem erro! Não sei se precisa alguma configuração!!!!! Agradeço a atenção!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @danilobicas-unoeste como vai?
Qual browser e IDE você está utilizando?

**Reply by danilobicas-unoeste** [reply]
VS Code 1.91.1 e o browser é o chrome mas não sei te falar a versão agora.. este problema deu no meu notebook de casa... No meu trabalho eu consigo debugar normalmente.... por isso minha dúvida sobre alguma configuração específica.... obrigado

**Comment by danilobicas-unoeste** [comment]
Balta.... Na verdade eu não estou conseguindo debugar o "WASM"! A API  debuga e para nos breakpoints sim.... Mas no webassembly (projeto web) tem aquele comportamento de ficar um círculo cinza no breakpoint e não para neles.

Obrigado

**Reply by andrebaltieri** [reply]
Tranquilo, isto acontece... depende da versão do Browser... mas aqui tem como resolver:
https://learn.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-8.0&tabs=visual-studio

**Comment by danilobicas-unoeste** [comment]
Muito obrigado. RESOLVIDO
Estava faltando configuração no Properties ->launchSettings.json no projeto web-wasm....
